pom.xml

```diff
@@ -1000,7 +1000,7 @@
             <!--Hamcrest first, then JUnit, then Mockito. See http://goo.gl/e5bJA5-->
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-all</artifactId>
-            <version>1.1</version>
+            <version>1.3</version>
             <scope>test</scope>
         </dependency>
         <dependency>
```

Comment:
Is there a reason not to update to junit 4.12, mockito 1.10.19, and xmlunit 1.6?  It looks like those are part of this ticket.  Are those updates going to be in future PRs?